### ISSUE TITLE
Cannot push function

### ISSUE THREAD

**Issue by phts** [LivePersonInc/faas-cli#42]
push always returns error "Lambda was modified already":

```
$ node_modules/.bin/lpf login --accountId xxx --username xxx --password xxx --userId xxx --token xxx
Welcome to
  _     ____    _____                 _   _                
 | |   |  _ \  |  ___|   _ _ __   ___| |_(_) ___  _ __  ___
 | |   | |_) | | |_ | | | | '_ \ / __| __| |/ _ \| '_ \/ __|
 | |___|  __/  |  _|| |_| | | | | (__| |_| | (_) | | | \__ \
 |_____|_|     |_|   \__,_|_| |_|\___|\__|_|\___/|_| |_|___/
                                                           
Use lpf help or lpf <command> --help for further informations.

$ node_modules/.bin/lpf push xxx                                                            
? Do you want to approve and overwrite the following lambda?
Caution: This action can NOT be reverted!

 
  AccountId:              xxx
  Name:                   xxx
  Description:            xxx.
  Event:                  conversational_command
  Dependencies:           -
  Environment variables:  -

   Yes

Pushing following functions:

  ✖ Pushing getOracleTicket
    → Lambda was modified already
```

**Comment by troykirin** [comment]
I am also experiencing the same issue as @phts

Logged in prior to pushing with both methods
1. `lpf login -a $accountId -u $email -p $password`
2. `lpf login --token $token --accountId $accountId --userId $userId`

**Comment by chrbrt** [comment]
fixed with the latest version (1.28.0) please use ```npm i -g liveperson-functions-cli```to update the cli
